index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/6d267a36ba.js" crossorigin="anonymous"></script>
+    <title>Portfolio Template</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <section class="main">
+        <div>
+            <nav>
+                <ul class="menu">
+                    <li>Portfolio</li>
+                    <li>About</li>
+                    <li>Contact</li>
+                </ul>
+            </nav>
+            <div class="h-menu">
+                <h1>Welcome</h1>
+                <div>
+                <div class="bar"></div>
+                <div class="bar"></div>
+                <div class="bar"></div>
+                </div>
```

Comment:
- [ ] This hamburger menu button is part of your navigation especially on small devices as the case is now and since it is for navigation, don't you agree that it should be nested in a `nav` tag? Let's place it in a `nav` tag please. This will be useful for accessibility ratings and improves user experience. :wink: